### PR TITLE
Fix back button style on login page

### LINKED ISSUE #23775: Enable to back to shop in admin login page
## Is your feature request related to a problem?

when you arrive on the admin page by mistake, it will be interesting to have the possibility of coming back to the site (this often happens for administrators.)

## Describe the solution you'd like

Enable to back to shop in admin login page.
Only the “back to my shop” will be changed and placed on the top left corner of the page:

image

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Back button wasn't well styled on the login page
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #23775.
| How to test?      | Go on login page and see the back button, test on mobile size too
| Possible impacts? | Back button on login page

## Screenshots
image
image

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/28235)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by HanaRebaiQA — COMMENTED** [review]
Hello @NeOMakinG 

I have checked this PR. I found these issues :

1. When the Shop is in RTL languages (As Arabic), then log out, the arrow is reversed in the desktop and mobile

**With PR** 
image
image

**Without PR** : The arrow is well displayed with RTL
image


2. When putting the mouse to click on the shop, the whole sentence with the arrow is underlined

**With PR**
image
image

https://user-images.githubusercontent.com/92912932/163186898-d74221e0-eb5a-412c-8ab3-42049848f8ad.mp4

**Without PR**
image

https://user-images.githubusercontent.com/92912932/163187009-15f0eb0a-ce80-4fa1-8c81-57506d83bfcd.mp4


3. The font-size is too bigger 

**With PR**
image
image


**Without PR**
image


The result if the `font-size` is updated to `12px` will be like this (I just change the font-size)
image


@TristanLDD : What do you think about the font-size ?

@NeOMakinG Could you please check it ?

Thanks!

**Comment by NeOMakinG** [comment]
> Hello @NeOMakinG
> 
> I have checked this PR. I found these issues :
> 
> 1. When the Shop is in RTL languages (As Arabic), then log out, the arrow is reversed in the desktop and mobile
> 
> **With PR** image image
> 
> **Without PR** : The arrow is well displayed with RTL image
> 
> 2. When putting the mouse to click on the shop, the whole sentence with the arrow is underlined
> 
> **With PR** image image
> 
>  all.selected.mp4 
> **Without PR** image
> 
>  without.PR.mp4 
> 3. The font-size is too bigger
> 
> **With PR** image image
> 
> **Without PR** image
> 
> The result if the `font-size` is updated to `12px` will be like this (I just change the font-size) image
> 
> @TristanLDD : What do you think about the font-size ?
> 
> @NeOMakinG Could you please check it ?
> 
> Thanks!

Fixed already

**Review by matks — DISMISSED** [review]

File: admin-dev/themes/default/template/controllers/login/content.tpl [hunk 1]
```diff
@@ -164,8 +164,11 @@
 		</div>
 		{/if}
 	</div>
-    <p class='text-center'><i class="icon-caret-left"></i> {l s='Back to' d='Admin.Actions'} <a href='{$homeUrl}'>{$shop_name}</a></p>
+
+  <a class='login-back' href='{$homeUrl}'><i class="material-icons">{if $lang_is_rtl}arrow_forward{else}arrow_back{/if}</i> <span>{l s='Back to' d='Admin.Actions'}</span> <span class="login-back-shop">{$shop_name}</span></a>
```
Comment (mparvazi) [inline admin-dev/themes/default/template/controllers/login/content.tpl]:
```suggestion
   <a class='login-back' href='{$homeUrl}'><i class="material-icons rtl-flip">arrow_back</i> <span>{l s='Back to' d='Admin.Actions'}</span> <span class="login-back-shop">{$shop_name}</span></a>
```

**Comment by mparvazi** [comment]
Just a suggestion:
Removing extra space in bottom of front panel.

Screenshot from 2022-04-16 18-49-50

Screenshot from 2022-04-16 18-47-23

**Comment by NeOMakinG** [comment]
@mparvazi applied you changes

**Review by Progi1984 — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Review by atomiix — DISMISSED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by HanaRebaiQA — APPROVED** [review]
Hello @NeOMakinG 

I checked again this PR. All issues are fixed.
For the `font-size`, Thanks to @TristanLDD who confirmed to me that it's `16px`.

So, here my tested cases : 
-Check with desktop : Ok 
image

-Check with mobile : OK
image

-Check with responsive 
image


-Check with different languages (Spanish and RTL language) : OK
image
image

-Check with different browsers (Chrome & Firefox) : Ok
image
image
image

-Check when configure multistore : OK

https://user-images.githubusercontent.com/92912932/165492503-86c016e3-bf78-4676-9e83-b3bca6de4f34.mp4


So, it's QA :heavy_check_mark: 

Thanks!

**Comment by NeOMakinG** [comment]
Thanks for testing @HanaRebaiQA

**Pull request merged**